e2e_tests/integration/connect-form.spec.ts

```diff
@@ -76,10 +76,15 @@ describe('Connect form', () => {
     getBoltSchemeSelect().should('have.value', aliasScheme)
   })
 
-  it('can connect with bolt:// protocol', () => {
+  it.only('can connect with bolt:// protocol', () => {
```

Comment:
Was this intentional?

Reply:
No, I missed it after debugging the test. Should have read over the files one more time, good catch!